libs/service-portal/vehicles/src/screens/VehicleDetail/VehicleDetail.tsx

```diff
@@ -106,19 +106,31 @@ const VehicleDetail: ServicePortalModuleComponent = ({ userInfo }) => {
 
         <UserInfoLine
           label={formatMessage(messages.capacity)}
-          content={mainInfo?.cubicCapacity?.toString() + ' cc.'}
+          content={
+            mainInfo?.cubicCapacity
+              ? mainInfo?.cubicCapacity?.toString() + ' cc.'
```

Comment:
You are already checking if the property is set so optional chaining is unnecessary.
```suggestion
              ? mainInfo.cubicCapacity.toString() + ' cc.'
```